### PR TITLE
Automate setup

### PR DESCRIPTION
**Pull request by MattSurabian**
~~This still needs documentation, but~~ The dynamic inventory and playbook included here allow everything that's necessary to control the entire class while still being able to target individual users machines without needing to modify any committed inventory file.

~~I'll come back to this later in the weekend or early next week with docs.~~

ToDo:
  - [x] Commit a private key to the repo and setup a single learndeployment user for everyone to login to. Having everyone log in as `ubuntu` prevented instructors from overriding invalid configurations with subsequent ansible runs.

  - [x] Document usage of dynamic inventory in instructor notes

### CODE REVIEW

**Comment by MattSurabian** [comment]
Also, due to the dynamic inventory file this class is will be able to be launched in an AZ occupied with other hosts without risk of unintended consequences.

**Comment by MattSurabian** [comment]
Ready for review!

**Comment by tkellen** [comment]
Looking forward to reviewing this! It's going to have to wait until next week tho :(

File: workshop/exercises/ssh-basics/overview.md [hunk 1]
```diff
@@ -69,4 +69,4 @@ ssh -i /path/to/key workshop.learndeployment.com
 
 This exercise comes with a private key. Use it in conjunction with the `ssh`
 command to connect to the server running at `workshop.learndeployment.com`.
-You will need to connect as the user `ubuntu`.
+You will need to connect as the user ``.
```
Comment (tkellen) [inline workshop/exercises/ssh-basics/overview.md]: the user `workshop`?

**Comment by tkellen** [comment]
:+1: sans that minor nit about the ssh-basics user.  I'll fix that after this merge.

**Pull request merged**